src/lib/Response/NoRenderResponse.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\HybridPlatformUi\Response;
+
+/**
+ * Interface to indicate that this response should bypass the app renderer.
```

Comment:
With `isRedirect()` implemented as suggested above that interface isn't necessary anymore.